src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandOddsRatio.java

```diff
@@ -20,37 +20,100 @@
  *
- * <p>Strand bias is a type of sequencing bias in which one DNA strand is favored over the other, which can result in incorrect evaluation of the amount of evidence observed for one allele vs. the other. The StrandOddsRatio annotation is one of several methods that aims to evaluate whether there is strand bias in the data. It is an updated form of the Fisher Strand Test that is better at taking into account large amounts of data in high coverage situations. It is used to determine if there is strand bias between forward and reverse strands for the reference or alternate allele.</p>
+ * <p>Strand bias is a type of sequencing bias in which one DNA strand is favored over the other, which can result in
+ * incorrect evaluation of the amount of evidence observed for one allele vs. the other. The StrandOddsRatio annotation
+ * is one of several methods that aims to evaluate whether there is strand bias in the data. It is an updated form of
+ * the <a href="https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_hellbender_tools_walkers_annotator_FisherStrand.php">Fisher Strand Test</a>
+ * that is better at taking into account large amounts of data in high coverage situations. It is used to determine if
+ * there is strand bias between forward and reverse strands for the reference or alternate allele(s).</p>
  *
  * <h3>Statistical notes</h3>
- * <p> Odds Ratios in the 2x2 contingency table below are</p>
  *
- * $$ R = \frac{X[0][0] * X[1][1]}{X[0][1] * X[1][0]} $$
- *
- * <p>and its inverse:</p>
+ * <p>The following 2x2 contingency table gives the notation for allele support and strand orientation.</p>
  *
  * <table>
- *      <tr><td>&nbsp;</td><td>+ strand </td><td>- strand</td></tr>
- *      <tr><td>REF;</td><td>X[0][0]</td><td>X[0][1]</td></tr>
- *      <tr><td>ALT;</td><td>X[1][0]</td><td>X[1][1]</td></tr>
+ * <tr><th>&nbsp;</th><th>+ strand&nbsp;&nbsp;&nbsp;</th><th>- strand&nbsp;&nbsp;&nbsp;</th></tr>
+ * <tr><th>REF&nbsp;&nbsp;&nbsp;</th><td>X[0][0]</td><td>X[0][1]</td></tr>
+ * <tr><th>ALT&nbsp;&nbsp;&nbsp;</th><td>X[1][0]</td><td>X[1][1]</td></tr>
  * </table>
  *
- * <p>The sum R + 1/R is used to detect a difference in strand bias for REF and for ALT (the sum makes it symmetric). A high value is indicative of large difference where one entry is very small compared to the others. A scale factor of refRatio/altRatio where</p>
+ * <p>We can then represent the Odds Ratios with the equation:</p>
+ *
+ * <img src="http://latex.codecogs.com/svg.latex?$$ R = \frac{X[0][0] * X[1][1]}{X[0][1] * X[1][0]} $$" border="0"/>
+ *
+ * <p>and its inverse:</p>
+ *
+ * <img src="http://latex.codecogs.com/svg.latex?$$ \frac{1}{R} = \frac{X[0][1] * X[1][0]}{X[0][0] * X[1][1]} $$" border="0"/>
  *
- * $$ refRatio = \frac{max(X[0][0], X[0][1])}{min(X[0][0], X[0][1} $$
+ * <p>The sum R + 1/R is used to detect a difference in strand bias for REF and for ALT. The sum makes it symmetric.
```

Comment:
I wouldn't say that the sum makes it symmetric.  That name persists from early stages of development when the model was different.  It's actually important to note that it's NOT symmetric because we don't want to penalize sites where the alt allele looks well balanced, but the ref allele isn't, presumably by chance.

Reply:
I didn't write these parts you are commenting on @ldgauthier and assume whoever did knew what they were talking about. I am hesitant to change/remove parts without discussing with the originator.

Reply:
Bertrand wrote this part of the comments (https://github.com/broadinstitute/gsa-unstable/commit/fb3d3e84e4951d3ea9e3a93d0ffb1559b9004b4e), but that was before the calculation was updated with the altRatio (which was my change: https://github.com/broadinstitute/gsa-unstable/commit/893997ddb5c0456714ba782934c3478010da1c09 -- apparently the "ensures that the annotations is large only" was my fault, but it was five years ago.)